numba_dppy/extended_numba_itanium_mangler.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from numba.core import itanium_mangler, types
+from numba_dppy import target
+
+
+def mangle_type_or_value(typ):
+    """
+    Mangle type parameter and arbitrary value.
+
+    This function extends Numba's `magle_type_or_value()` to
+    support numba.types.CPointer type.
```

Comment:
` This function extends Numba's ``magle_type_or_value()`` to
    support numba.types.CPointer type, e.g., an ``int *`` argument will be mangled to "Pi".`

Double back tick helps it render correctly in Sphinx (maybe we are missing some extension and that is why double back tick is needed). The other is just grammar nitpick.

Reply:
Updated